pkg/apis/serving/v1alpha1/kfservice_framework_tensorrt.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2019 kubeflow.org.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+var (
+	defaultTensorRTISImageName = "nvcr.io/nvidia/tensorrtserver"
```

Comment:
Can you also add the config entry in these two places? 
https://github.com/kubeflow/kfserving/blob/master/pkg/apis/serving/v1alpha1/framework.go#L63
https://github.com/kubeflow/kfserving/blob/master/config/default/configmap/kfservice.yaml